docs/en/operations/settings/merge-tree-settings.md

```diff
@@ -1133,10 +1133,24 @@ By using `ORDER BY time DESC` in the query, `ReadInOrder` is applied.
 
 ## cache_populated_by_fetch
 
-A Cloud only setting. 
+:::note
+This setting applies only to ClickHouse Cloud.
+:::
 
 When `cache_populated_by_fetch` is disabled (the default setting), new data parts are loaded into the cache only when a query is run that requires those parts.
 
-If enabled, `cache_populated_by_fetch` will instead cause all nodes to load new data parts from storage into their cache without requiring a query to trigger such an action. 
+If enabled, `cache_populated_by_fetch` will instead cause all nodes to load new data parts from storage into their cache without requiring a query to trigger such an action.
 
 Default value: 0.
+
+## enable_minmax_index_for_all_numeric_columns
+
+When enabled, min-max (skipping) indices are added for all numeric columns of the table.
+
+Default value: false.
```

Comment:
Question: Can we enable both settings either in this PR or a follow-up PR (*)? Min-max indices are lightweight and most analytics databases have them enabled by default.

(*) Performance of INSERTs may be impacted by enabling the settings and tests may need to be fixed, so it preferred to enable the settings in a follow-up.

Reply:
ya, sounds good. Will make a follow-up PR to enable them by default